src/Microsoft.TemplateEngine.Cli/Microsoft.TemplateEngine.Cli.csproj

```diff
@@ -6,6 +6,7 @@
     <IsShippingPackage>false</IsShippingPackage>
     <EnablePublicApiAnalyzer>true</EnablePublicApiAnalyzer>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Can we just enable it globally in root Directory.props?

Reply:
I wish I tried that and failed a bit, problem is that if we enable it for NET472 projects things fail to compile and stuff, maybe some other day, just not today...